pkg/questionModel/plainText.go

```diff
@@ -0,0 +1,101 @@
+package questionModel
+
+import (
+	"errors"
+	"fmt"
+	"simple-ec2/pkg/ec2helper"
+	"strings"
+
+	"github.com/charmbracelet/bubbles/textinput"
+	tea "github.com/charmbracelet/bubbletea"
+)
+
+type PlainText struct {
+	textInput         textinput.Model
+	question          string
+	validFunctions    []CheckInput
+	EC2Helper         *ec2helper.EC2Helper
+	invalidMsg        string
+	displayInvalidMsg bool
+	err               error
+}
+
+func (pt *PlainText) InitializeModel(data *BubbleTeaData) {
+	ti := textinput.New()
+	ti.Placeholder = data.DefaultOption
+	ti.Focus()
+	ti.CharLimit = 156
+	ti.Width = 40
```

Comment:
Where does this width value come from?

Reply:
I made it up. It was originally 20, but I made it larger because I put in test values and 20 seemed too short

Reply:
For exmaple an AMI was longer than 20 characters

Reply:
Im actually removing it because removing it makes it so theres no limit